src/BatteryStateOfChargeService.h

```diff
@@ -45,5 +45,10 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
 
+    double AmpHoursUsed_;
+    double newCurrent_;
+    double sumCurrent_;
```

Comment:
This variable is not really being used for anything productive, it's just being updated, should probably delete it since it's not being used.